src/codecoverage/bot/code_coverage_bot/uploader.py

```diff
@@ -96,13 +96,40 @@ def check_codecov_job():
         return False
 
 
-def gcp_upload(path, data):
+def gcp(repository, revision, data):
     '''
-    Upload a data payload on Google Cloud Storage
+    Upload a grcov raw report on Google Cloud Storage
+    * Compress with gzip
```

Comment:
It would be nice to use zstandard instead of gzip.
Especially for the speed of decompression, which might matter on the backend.

Reply:
Done :boom: